contributing/code/standards.rst

```diff
@@ -147,9 +147,7 @@ Structure
 
 * Always use `identical comparison`_ unless you need type juggling;
 
-* Use `Yoda conditions`_ when checking a variable against an expression to avoid
-  an accidental assignment inside the condition statement (this applies to ``==``,
-  ``!=``, ``===``, and ``!==``);
+* Use your IDE to check for Assignment In Conditions and make sure it highlights it as an error.;
```

Comment:
No need for a dot _and_ semicolon, a `;` in a sentence indicates a break but the subject is not stopped as done with a period. Using only `;` is enough.